### PR TITLE
chore(storage): RiseCtl Support more concise Version printing

### PR DESCRIPTION
**Pull request by Li0k**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

- Metrics explicitly distinguish between sub_level type
image

- RiseCtl Support more concise version printing
In my scenario, the entire version is promised to be unreadable, so I support a more concise and readable printout on risectl to quickly determine the level type and sst information for lsm.

image

## Checklist For Contributors

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [ ] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

**Review by soundOfDestiny — APPROVED** [review]
LGTM. Debug info is welcome!

File: src/ctl/src/lib.rs [hunk 1]
```diff
@@ -76,7 +76,10 @@ enum ComputeCommands {
 #[derive(Subcommand)]
 enum HummockCommands {
     /// list latest Hummock version on meta node
-    ListVersion,
+    ListVersion {
+        #[clap(short, long = "full-print", default_value_t = false)]
+        full_print: bool,
```
Comment (hzxa21) [inline src/ctl/src/lib.rs:81]: nits: by convention, it is better call it `--verbose` (`-v`) to print more info.

**Review by hzxa21 — APPROVED** [review]
+1. This is super useful. Debug info is oftenly missing.

File: src/ctl/src/lib.rs (hunk 1 above)
Comment (Li0k) [reply src/ctl/src/lib.rs]: fix it

**Pull request merged**